WireDomain/Sources/WireDomain/Repositories/Client/ClientRepositoryError.swift

```diff
@@ -0,0 +1,21 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+/// Errors originating from `ClientRepository`.
+
+enum ClientRepositoryError: Error {}
```

Comment:
Is it necessary now?

Reply:
removed in https://github.com/wireapp/wire-ios/pull/2019/commits/d0e4a57aa3390d8146ceda7f9ce8dd4ac87cf0b4